ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -21,21 +21,21 @@
 </div>
 
 <div class="mt-8 prose prose-lg text-gray-800 pb-6">
-  <h3 class="ds-h3">Examples of Correct use</h3>
+  <h3 class="ds-h3">Examples of correct use</h3>
   <CodeBlock @language="markup" @code="&lt;button aria-label=&quot;Check Activity&quot;>
  &lt;FlightIcon @name=&quot;activity&quot; />
 &lt;/button>" />
   <CodeBlock @language="markup" @code="&lt;h2>Activity Report &lt;FlightIcon @name=&quot;activity&quot; />&lt;/h2>" />
 </div>
 
 <div class="mt-8 prose prose-lg text-gray-800 pb-6">
-  <h2 class="ds-h2" id="ember-flight-icons"><a href="#ember-flight-icons" class="ds-a" rel="external">&sect;</a> Use in Ember Apps</h2>
+  <h2 class="ds-h2" id="ember-flight-icons"><a href="#ember-flight-icons" class="ds-a" rel="external">&sect;</a> Use in Ember apps</h2>
   <h3 class="ds-h3">Installation</h3>
   <p>To install, run
   <CodeBlock @code="yarn add @hashicorp/ember-flight-icons" @language="bash" />
   </p>
   <p>Note: <kbd class="ds-kbd">ember-test-selectors</kbd> is a dependency added for the author's convenience.
-  <a href="https://github.com/simplabs/ember-test-selectors" class="ds-a">This Ember Addon</a> strips out all <kbd class="ds-kbd">data-test-*</kbd> attributes for production builds.</p>
+  <a href="https://github.com/simplabs/ember-test-selectors" class="ds-a">This Ember package</a> strips out all <kbd class="ds-kbd">data-test-*</kbd> attributes for production builds.</p>
```

Comment:
issue: I'd like us to stick to standard Ember nomenclature; I think we decided with @didoo that if we explicitly specified "Ember Addon" then saying "addon" vs "package" or "library" was okay to do.

Reply:
no opinion but i thought didoo said the opposite, let's document in contributing once it's decided?

Reply:
I am ok to call "addon" when is associated with Ember (so "Ember addon" is OK for me, not sure if upper/lowecase "a", let's use the Ember convention). I was more keen to avoid to use "addon" as a standalone word when we mean JavaScript/npm package. Makes sense?